network-api/networkapi/tests.py

```diff
@@ -465,3 +467,136 @@ def test_software_page_export_fields(self):
+        # Test fields before the command is run
+        # Given a page has a cloudinary image,
+        # When converting cloudinary images to wagtail images,
+        # Then wagtail images should be the same image as the old cloudinary image
+        # Ensure they have cloudinary fields
+        software_product_page = Page.objects.get(pk=self.software_product_page.id).specific
+        general_product_page = Page.objects.get(pk=self.general_product_page.id).specific
+        # Assert there are cloudinary images that will be converted to Wagtail images.
+        self.assertTrue(hasattr(software_product_page, 'cloudinary_image'))
+        self.assertTrue(hasattr(general_product_page, 'cloudinary_image'))
+        # Assert General Product Page fields are what we expect when setting them in the setUp() method
+        self.assertEqual(general_product_page.offline_capable, 'Yes')
+        self.assertEqual(
+            general_product_page.offline_use_description,
+            'Although it is unclear how offline capabilities work'
+        )
+        # Assert Software Product Page fields are what we expect when setting them in the setUp() method
+        self.assertEqual(software_product_page.recording_alert, 'Yes')
+        self.assertEqual(software_product_page.handles_recordings_how, 'Sample software recording description')
+        # Assert the image field and cloudinary fields are not the same
+        self.assertNotEqual(
+            software_product_page.image.filename,
+            f'{software_product_page.cloudinary_image.public_id}.{software_product_page.cloudinary_image.format}',
+        )
+        self.assertNotEqual(
+            general_product_page.image.filename,
+            f'{general_product_page.cloudinary_image.public_id}.{general_product_page.cloudinary_image.format}',
+        )
+
+        # Run the management command which will convert Cloudinary images to Wagtail images.
+        call_command("migrate_cloudinary")
+
+        # Test fields after the management command is run
+        # Re-fetch the page data from the database.
+        software_product_page = Page.objects.get(pk=self.software_product_page.id).specific
+        general_product_page = Page.objects.get(pk=self.general_product_page.id).specific
+        self.assertTrue(hasattr(software_product_page, 'image'))
+        self.assertTrue(hasattr(general_product_page, 'image'))
+        # Assert General Product Page "specific" fields have not changed
+        self.assertEqual(general_product_page.offline_capable, 'Yes')
+        self.assertEqual(
+            general_product_page.offline_use_description,
+            'Although it is unclear how offline capabilities work'
+        )
+        # Assert Software Product Page "specific" fields have not changed
+        self.assertEqual(software_product_page.recording_alert, 'Yes')
+        self.assertEqual(software_product_page.handles_recordings_how, 'Sample software recording description')
```

Comment:
And then here, we can iterate through the previous created dicts, where we verify every field with an skip for the image fields.